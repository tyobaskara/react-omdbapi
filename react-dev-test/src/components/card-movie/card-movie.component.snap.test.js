import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import CardMovie from './card-movie.component';

test('CardMovie snaptest', () => {
  const props = {
    imdbID: '123',
    Poster: 'Poster',
    Title: 'Title',
    Type: 'Type',
    Year: 'Year',
    onClick: () => {},
    match: {
      path: 'path'
    },
  }
  const component = renderer.create(
    <BrowserRouter>
      <CardMovie {...props} />
    </BrowserRouter>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});