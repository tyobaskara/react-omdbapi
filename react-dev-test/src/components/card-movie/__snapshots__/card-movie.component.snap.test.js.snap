// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardMovie snaptest 1`] = `
<li
  className="card-movie"
>
  <div
    className="content-wrap"
  >
    <div>
      <img
        alt="Title"
        className="poster"
        src="Poster"
      />
    </div>
    <h2>
      Title
    </h2>
    <ul
      className="separating-info"
    >
      <li>
        Type
      </li>
      <li>
        Year
      </li>
    </ul>
    <a
      className="link-more-info"
      href="//123"
      onClick={[Function]}
    >
      More Info
    </a>
  </div>
</li>
`;
